src/interfaces/InjectedSenderWallet.ts

```diff
@@ -101,16 +91,15 @@ export interface RequestSignTransactionsParams {
 }
 
 interface InjectedSenderWallet {
-  init: (params: InitParams) => Promise<InitResponse>;
+  isSender: string,
```

Comment:
I believe this property returns `boolean`

Reply:
yes. thanks.